### ISSUE TITLE
Invariant Violation: Minified React error #31 - with the version 3.0.0-rc.9

### ISSUE THREAD

**Issue by manikumarrepuri** [appbaseio/reactivesearch#986]
**Issue Type:**
Seems to be a Bug

**Platform:**
Web

**Description:**
react-dom.production.min.js:4408 Invariant Violation: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Bvalue%2C%20relation%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

An error has occured. You're using Reactivesearch Version: 3.0.0-rc.9. If you think this is a problem with Reactivesearch, please try updating to the latest version. If you're already at the latest version, please open an issue at https://github.com/appbaseio/reactivesearch/issues

**Comment by jyash97** [comment]
Can you please share a codesandbox link. Looks like you are rendering an `Object` somehwhere.

**Comment by manikumarrepuri** [comment]
I got what this issue is all about. 
In the elasticsearch, there is a change recently in which the total was changed from number to an object. Below is the reference....
{
     "_shards": ...
     "timed_out": false,
     "took": 100,
     "hits": {
         "max_score": 1.0,
         "total" : {
             "value": 10000,    
             "relation": "gte"  
         },
         "hits": ...
     }
}

In the above code total has been changed to an object, which is throwing as a rendering issue under ReactiveList. It was a number before.
This is resolved by keeping rest_total_hits_as_int: true as one of the search property.

I feel ReactiveList point of view, this has to be considered as it's been changed in the recent ES release.
